src/Misc/default-file-browsing.html.pode

```diff
@@ -0,0 +1,225 @@
+<html lang="en" ; style='background-color: #01b0c4' ;>
+
+<head>
+    <!-- Include the Bootstrap Icons stylesheet -->
+    <link rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-icons/1.8.1/font/bootstrap-icons.min.css">
+
+    <title>File Browser:$($Data.Path)</title>
+    <meta charset='utf-8'>
+
+    <style>
+        h3 {
+            font-family: 'Courier New', Courier, monospace;
+            /* Applies monospace font to <h2> */
+        }
```

Comment:
There aren't any `h3` elements on the page for this to apply, however if we replace `h3` with `body` this will match the styling with other default Pode pages, and also won't affect the monospaced table either.

The font-family will also need replacing with `Helvetica,Arial,sans-serif`.